src/v2/guide/index.md

```diff
@@ -47,58 +47,58 @@ var app = new Vue({
 var app = new Vue({
   el: '#app',
   data: {
-    message: 'Hello Vue!'
+    message: 'Hello Vue !'
   }
 })
 </script>
 {% endraw %}
 
-We have already created our very first Vue app! This looks pretty similar to just rendering a string template, but Vue has done a lot of work under the hood. The data and the DOM are now linked, and everything is now **reactive**. How do we know? Just open your browser's JavaScript console (right now, on this page) and set `app.message` to a different value. You should see the rendered example above update accordingly.
+Nous venons tout juste de créer notre première application Vue ! Ça ressemble sacrément à du simple rendu dans un template, mais Vue effectue un réel travail dans l'ombre. Les données et le DOM sont maintenant couplés, et tout est à présent **réactif**. Comment s'en rendre compte ? Ouvrez juste la console JavaScript de votre navigateur (là maintenant, sur cette page) et attribuez à `app.message` différentes valeurs. Vous devriez voir le rendu de l'exemple en cours se mettre à jour en concordance.
```

Comment:
Ok, le boss final. Je suis complètement d'accord sur le fait que « dans l'ombre » colle pas. Je suis également d'accord sur le fait que « sous le capot » semble une traduction dans le même sens mais... je trouve que « sous le capot » fait perdre cette notion de « magie », « fantastique » qu'à « under the hood ». Genre « sous le capot » on voit très clairement ce qui se passe et la la phrase sous entend que « haha, ça à l'air magique mais il se passe quelque chose en coulisse... ».

Tiens, que direz vous de « mais Vue effectue un réel travail en coulisse » ?